packages/components/src/components/radio-group/radio-group.stories.ts

```diff
@@ -41,280 +48,97 @@ export default {
+      <sd-radio-group name="radio-group" required>
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+        <label slot="label">Required Group</label>
+      </sd-radio-group>
+    </div>
+  `
 };
 
 /**
- * `sd-radio-group` with multiple `sd-radio-buttons` is fully accessibile via keyboard.
+ * Use the `orientation` attribute to set the orientation of the radio buttons.
  */
 
-export const MouselessWithRadioButtons = {
-  parameters: { controls: { exclude: ['default', 'size', 'value', 'required', 'invalid', 'label', 'orientation'] } },
-  render: (args: any) => {
-    return html`<div class="mouseless">
-      ${generateTemplate({
-        args,
-        constants: [
-          {
-            type: 'slot',
-            name: 'default',
-            value:
-              '<sd-radio-button value="1"><sd-icon library="global-resources" name="system/picture" slot="icon"></sd-icon></sd-radio-button><sd-radio-button value="2"><sd-icon library="global-resources" name="system/picture" slot="icon"></sd-icon></sd-radio-button><sd-radio-button value="3"><sd-icon library="global-resources" name="system/picture" slot="icon"></sd-radio-button>'
-          },
-          {
-            type: 'attribute',
-            name: 'value',
-            value: `2`
-          },
-          {
-            type: 'slot',
-            name: 'label',
-            value: ``
-          }
-        ]
-      })}
-    </div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelectorAll('.mouseless sd-radio-button')[1];
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
-    el?.shadowRoot?.querySelector('button')!.focus();
-  }
+export const Orientation = {
+  name: 'Orientation',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio-group name="radio-group" orientation="vertical">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+        <label slot="label">Vertical Group</label>
+      </sd-radio-group>
+
+      <sd-radio-group name="radio-group" orientation="horizontal">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+        <label slot="label">Horizontal Group</label>
+      </sd-radio-group>
+    </div>
+  `
 };
 
 /**
- * 1. You can use the `setCustomValidity` method to set a custom validation message. This will override any native validation messages.
- * 2. Set an empty string to clear the custom validity and make the input valid.
- * 3. To show the validation message, call the `reportValidity` method. Originally this would show a native validation bubble, but we show the error messages inline.
+ * Use the `boldlabel` attribute to set the group label to bold.
```

Comment:
I’m confused about this attribute ... why do we have that? Group label is always bold ... there should not be an attribute for that.

Reply:
I would like to remove the attribute.

Reply:
Hidden as agreed in our daily sync and created a [ticket](https://github.com/solid-design-system/solid/issues/1274) to address it

Reply:
+1
I was shocked, that it exists. 😂